config.js

```diff
@@ -2,6 +2,9 @@
 
 (function () {
   require('./lib/main').config(
-    require('./lib/cli-options')(process.argv)
+    {
+      ...require('./lib/cli-options')(process.argv),
+      ...require('./lib/env-options')()
```

Comment:
This syntax might be breaking Node 6?

Will want to flip order so CLI always takes precedence

Reply:
Thanks @maxbeatty you're right, even though Node 6 supports es6 it does not support the spread operator on objects, only arrays. I'll update and flip precedence.

Reply:
strange that it worked when I tried it locally with same version of node though

Reply:
changes ready for review.